github/code-scanning.go

```diff
@@ -18,11 +18,17 @@ import (
 // GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/code-scanning/
 type CodeScanningService service
 
+type Tool struct {
+	Name    *string `json:"name,omitempty"`
+	GUID    *string `json:"guid,omitempty"`
+	Version *string `json:"version,omitempty"`
+}
+
 type Alert struct {
```

Comment:
```suggestion
// Alert represents an individual Github Code Scanning Alert on a single repository
//
// GitHub API docs: https://docs.github.com/en/rest/reference/code-scanning#list-code-scanning-alerts-for-a-repository
type Alert struct {
```

Reply:
In addition to the ending period, please spell "GitHub" with a capital `G` and capital `H`.  Thanks.